test/e2e/storage/volume_provisioning.go

```diff
@@ -1013,36 +1054,6 @@ func updateDefaultStorageClass(c clientset.Interface, scName string, defaultStr
 	verifyDefaultStorageClass(c, scName, expectedDefault)
 }
 
-func getClaim(claimSize string, ns string) *v1.PersistentVolumeClaim {
```

Comment:
these two were literally adjacent to each other in the same file? Golden.

Reply:
fun times